pyVoIP/sock/sock.py

```diff
@@ -182,11 +183,18 @@ def __init__(
                 )
             self.s = self.server_context.wrap_socket(self.s, server_side=True)
 
-        self.buffer = sqlite3.connect(
-            pyVoIP.SIP_STATE_DB_LOCATION,
-            isolation_level=None,
-            check_same_thread=False,
-        )
+        if sys.version_info.major == 3 and sys.version_info.minor >= 12:
+            self.buffer = sqlite3.connect(
+                pyVoIP.SIP_STATE_DB_LOCATION,
+                check_same_thread=False,
```

Comment:
No ``isolation_level=None,`` here?

Reply:
No, it's deprecated as of Python 3.12, we're supposed to use `autocommit` instead